model-optimizer/mo/front/mxnet/loader.py

```diff
@@ -106,12 +109,25 @@ def symbol2nx(graph, model_nodes, model_params, input_names: str = ''):
         graph.add_node(node_name, **symbol_attrs(node))
         graph.node[node_name].update(common_mxnet_fields(Node(graph, node_name)))
         index_node_keys[i] = node_name
+        fw_name_map[node_name] = node['name']
```

Comment:
Does it make sense to use node.soft_get('name', node.id)?

Reply:
Yes, I think it makes sense. But I think the the default value should be empty string, as node.id will be randomly generated id that does not correspond to the framework name.

Reply:
We can't use soft_get() here as node is a dictionary obtained from loading .json model file.